.docker.env.example

```diff
@@ -0,0 +1,42 @@
+# See server/config.js for details
+# Remember to only include the variables that you are overriding
```

Comment:
This is a serious project seen by millions. You need to be less friendly in your comments, whoever wrote this